easybuild/easyconfigs/p/Python/Python-2.7.8-foss-2014b.eb

```diff
@@ -89,9 +94,8 @@ exts_list = [
     ('pyparsing', '2.0.2', {
         'source_urls': ['https://pypi.python.org/packages/source/p/pyparsing/'],
     }),
-    ('pygments', '1.6', {
-        'source_urls': ['https://pypi.python.org/packages/source/P/Pygments/'],
-        'source_tmpl': 'Pygments-%(version)s.tar.gz',
+    ('matplotlib', '1.3.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/m/matplotlib/'],
```

Comment:
this shouldn't be added back, it was removed with a reason (extra deps required like libpng)